test/unit/subscription_mailer_test.rb

```diff
@@ -61,4 +61,23 @@ class SubscriptionMailerTest < ActionMailer::TestCase
     assert_equal "[PublicLab] #{user.username} liked your question", email.subject
     assert email.body.include?("Public Lab contributor #{user.username} (https://#{request_host}/profile/#{user.username}) just liked your question")
   end
+
+  test 'notify users who do not follow particular tag' do
+    node = node(:one)
+    node_tags = node.tags
+    new_tag = tags(:spam)
+    user = users(:spammer)
+    users_not_following_tags = new_tag.followers_who_dont_follow_tags(node_tags)
+    users_to_email = users_not_following_tags.reject { |u| u.uid == user.uid }
+    assert_difference 'ActionMailer::Base.deliveries.size', users_to_email.count do
```

Comment:
Here I think hard coding the exact figure may break the test in case a person decides to add more users to the fixtures. It is also more readable :smile: